root/io/compression/MainEvent.cxx

```diff
@@ -305,10 +305,14 @@ int main(int argc, char **argv)
       int expectedSize = -1;
       if (comp == 0) expectedSize = 5534559;
       else if (comp == 101) expectedSize = 1254808;
-      else if (comp == 201) expectedSize = 1111230;
+      else if (comp == 208) expectedSize = 1111230;
```

Comment:
Surprisingly the compression level change but not the final size, is that correct?

Reply:
Thank you, I updated references while running them zero tolerances.